source/includes/harvest/_users.md

```diff
@@ -126,3 +126,44 @@ employee_id | The Employee ID of the user to retrieve
 
 <br>
 [See noteworthy response attributes.] (#the-user-object)
+
+## POST: Create User
+
+```shell
+curl -X POST 'https://harvest.greenhouse.io/v1/users'
+-H "On-Behalf-Of: {greenhouse user ID}"
+-H "Authorization: Basic MGQwMzFkODIyN2VhZmE2MWRjMzc1YTZjMmUwNjdlMjQ6"
+```
+
+> The above command takes a JSON request, structured like this:
+
+```
+{
+  "first_name": "Bob",
+  "last_name": "Smith",
+  "email": "bob@email.org",
+  "send_email_invite": true
+}
+```
+
+Create a new user.
+
+### HTTP Request
+
+`POST https://harvest.greenhouse.io/v1/users`
+
+### Headers
+
+Header | Description
+--------- | -----------
+On-Behalf-Of | ID of the user issuing this request. Required for auditing purposes.
+
+### JSON Body Parameters
+
+Parameter | Required | Type | Description
+--------- | ----------- | ----------- | -----------
+first_name | Yes | string | The user's first name
+last_name | Yes | string | The user's last name
+email | Yes | string | The user's email address
+send_email_invite | No | boolean | If true, an email will be sent to the above email address inviting them to login. If false, nothing happens. Default is false.
```

Comment:
@tdphillipsjr Hmm, so the user can't login until they create a password or are configured? Is there any way for a user to do that without getting a link invitation? If there isn't, it makes me think that the default behavior is that an email should be sent. Does anyone know what the default behavior is in the UI?